scripts.js

```diff
@@ -13,50 +14,43 @@ addCopy.classList.add("fas", "fa-copy", "copy", "icon", "push");
 addDelete.classList.add("fas", "fa-times", "delete", "icon", "edit");
 
 // focus curson in input on load
-window.onload = getInput.focus();
+window.onload = inputElement.focus();
 
 // add return key listen and call addItem function
-getInput.addEventListener('keydown', function() {
+inputElement.addEventListener('keydown', function() {
   if (event.code == 'Enter') {
     addItem();
-    getInput.value = "";
-    getInput.focus();
+    inputElement.value = "";
+    inputElement.focus();
   }
 });
 
 // add event listeners
-function copyListener() {
-  const allCopyIcons = document.querySelectorAll('.copy');
-  if (allCopyIcons.length > 1){
-    allCopyIcons.forEach(e => e.addEventListener('click', copy));
-  } else {
-    allCopyIcons[0].addEventListener('click', copy)
-  };
-}
-
-function copy(e) {
-  const itemText = e.target.previousSibling.innerText;
+const copy = function(e) {
+  const itemText = e.target.previousElementSibling.innerText;
   itemText.select();
   document.execCommand("copy");
   console.log(itemText);
 }
 
-function deleteListener() {
-  const allDeleteIcons = document.querySelectorAll('.delete');
-  if (allDeleteIcons.length > 1){
-    allDeleteIcons.forEach(e => e.addEventListener('click', remove));
-  } else {
-    allDeleteIcons[0].addEventListener('click', remove)
-  };
+const remove = function(e) {
+  e.target.parentElement.remove();
 }
 
-function remove(e) {
-  e.target.parentElement.remove();
+const copyListener = function(addCopy) {
+    console.log(addCopy);
+    addCopy.addEventListener("click", copy)
 }
 
+const deleteListener = function(addDelete) {
+    console.log(addDelete);
+    addDelete.addEventListener("click", remove)
+}
+
+
 // add pasted item to list
 const addItem = function() {
```

Comment:
Haha. You've change this from:
```js
function addItem() { ... }
```
but you're still declaring it as a `function`, not an _arrow function_.

But this is my fault, I should have presented this differently.

Have a read here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions